lib/crawler/output_sink/elasticsearch.rb

```diff
@@ -43,9 +45,25 @@ def initialize(config)
         )
       end
 
-      def ping_output_index
-        raise Errors::IndexDoesNotExistError, system_logger.info("Failed to find index #{config.output_index}") unless
-          client.indices.exists(index: config.output_index)
+      def verify_es_connection
+        client.info
+      rescue StandardError # rescue because client.info crashes ungracefully when ES is unreachable
```

Comment:
Is it possible to catch more specific error here, rather than an umbrella "StandardError"?